R/data-objects.R

```diff
@@ -107,10 +200,10 @@ iput <- function(
     x[[3]]
   )
 
-  invisible(NULL)
 }
-# internal
-iput_ <- function(x, offset, count, logical_path, truncate, verbose) {
+# internal: object to irods conversion
+local_to_irods_ <- function(x, logical_path, offset, count, truncate, verbose)
```

Comment:
This could probably be merged with `local_to_irods()`, which is also internal, right?

Reply:
`local_to_irods()` is vectorised, whereas `local_to_irods_()` is not, so I think it is better to keep it separated.

Reply:
Ohh right, good point :)